remote-api/src/main/java/org/alfresco/rest/api/impl/rules/RuleSetsImpl.java

```diff
@@ -69,6 +76,34 @@ public RuleSet getRuleSetById(String folderNodeId, String ruleSetId, List<String
         return ruleSetLoader.loadRuleSet(ruleSetNode, folderNode, includes);
     }
 
+    @Override
+    public RuleSetLink linkToRuleSet(String folderNodeId, String linkToNodeId)
+    {
+
+        final NodeRef folderNodeRef = validator.validateFolderNode(folderNodeId,false);

```

Comment:
I think this should be true, not false.  I'm not sure, but I'd assume the user needs write access on the folder in order to create a child of it.